velocity.js

```diff
@@ -54,10 +63,10 @@ app.use('/materializecss', express.static(`${__dirname}/node_modules/materialize
 app.use('/animate', express.static(`${__dirname}/node_modules/animate.css/`));
 app.use(
     sassMiddleware({
-        src: `${__dirname}/sass`, 
+        src: `${__dirname}/sass`,
         dest: `${__dirname}/UI/stylesheets`,
-        prefix:  '/stylesheets',
-        debug: true, // TODO: remove before release
+        prefix: '/stylesheets',
+        debug: false, // TODO: remove before release
```

Comment:
why are we turning this off? lets have it on since we are still developing

Reply:
oooh yeah, sorry forgot to turn it back on. It spams the log files so I turned it off while fixing some issues. But yeah I will turn it back on